src/PanZoom.js

```diff
@@ -92,8 +117,8 @@ class PanZoom extends React.Component<Props> {
   frameAnimation = null
   intermediateFrameAnimation = null
 
-  transformMatrixString = null
-  intermediateTransformMatrixString = null
+  transformMatrixString = `matrix(0, 0, 0, 0, 0, 0)`
```

Comment:
The default value should be `matrix(1, 0, 0, 1, 0, 0)` as we still consider the default scale to be 1.